src/main/scala/firrtl/Emitter.scala

```diff
@@ -92,6 +92,14 @@ class VerilogEmitter extends Emitter {
     }
     case _ => error("Shouldn't be here")
   }
+  /** Turn Params into Verilog Strings */
+  def stringify(param: Param): String = param match {
```

Comment:
Similar to DefModule below, nothing is sealed, although since I'm adding this new IR node I think I will seal it